tests/cli/main.cpp

```diff
@@ -113,6 +114,30 @@ int get_available_port()
    return ntohs(sin.sin_port);
 }
 
+boost::filesystem::path create_api_access_file(fc::temp_directory& directory) {
+   boost::filesystem::path apiaccess_path = boost::filesystem::path{directory.path().generic_string()} / "api-access.json";
+   fc::path apiaccess_out = apiaccess_path;
+
+   const string apiaccess_content = R"(
+   {
+      "permission_map" :
+      [
+         [
+            "*",
+            {
+               "password_hash_b64" : "*",
+               "password_salt_b64" : "*",
+               "allowed_apis" : ["database_api", "network_broadcast_api", "history_api", "custom_operations_api"]
```

Comment:
Hm, I think the API should be allowed by default. Then you don't need to allow it specifically for the tests.

Reply:
Plug in is disabled by default, api can be enabled and check for plugin presence, fail it is not loaded?

Reply:
Can you only allow it if it's present? I thought there was no direct connection. (Didn't check though.)

Reply:
enabled api by default at https://github.com/bitshares/bitshares-core/pull/1926/commits/739bc780d8ba210141bfde1c1335db76ffd40ed9